index.html

```diff
@@ -30,33 +30,33 @@
         <div>
```

Comment:
*** Dear @brainskevm, I see some of your feature are already on the main branch, this is against the GitHub flaw. next time make sure to create a new branch and work from there for all the new feature. 

- [ ] this section is looking great @brainskev, But there are something's to improve. Make sure to use the right font type and size. 

https://github.com/brainskev/setup-and-mobile-version-skeleton/blob/7cd91a1185e09945d0ad9d85fbe02be92e70ef91/index.html#L145-L152

##Figma design 
![figma](https://user-images.githubusercontent.com/69077061/198847249-78c66741-d738-47c9-9285-0164169e436c.PNG)


## Your design 
![your](https://user-images.githubusercontent.com/69077061/198847260-f028a284-0811-4201-9ebe-4dd0415a48ee.PNG)


- [ ] also the `Let's connect` text should be wrapped with `hx` tag, this will give it semantic emphasis.
https://github.com/brainskev/setup-and-mobile-version-skeleton/blob/7cd91a1185e09945d0ad9d85fbe02be92e70ef91/index.html#L149